src/modules/assessments/components/AssessmentAutofillButton.tsx

```diff
@@ -0,0 +1,21 @@
+import { Button } from '@mui/material';
```

Comment:
Nothing new in this file, the button is just factored out into its own component so it can be used by both the `AssessmentForm` (where it's shown on mobile above the other input fields), and the `AssessmentFormSideBar` (which is only shown on desktop).